docs/pages/badge.js

```diff
@@ -25,51 +24,255 @@ export default function BadgePage({ generatedDocGen }: {| generatedDocGen: DocGe
           <MainSection.Card
             cardSize="md"
             type="do"
-            title="When to Use"
+            title="When to use"
             description={`
           - Labeling and bringing awareness to a specific element or feature (e.g., something is new or required).
         `}
           />
           <MainSection.Card
             cardSize="md"
             type="don't"
-            title="When Not to Use"
+            title="When not to use"
             description={`
           - Providing feedback at the element level (e.g., displaying error messages). Use inline text instead.
           - Requiring interaction from users since Badges are always static and non-interactive.
         `}
           />
         </MainSection.Subsection>
       </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Align the badge to the top of large accompanying text. Center align for standard size text."
+            defaultCode={`
+              <Text size="600">Ads & Campaigns <Badge text="Beta" position="top"/></Text>
+            `}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use colored badges over media. Instead use the dark or light wash."
+            defaultCode={`
+            <Mask height={150} width={280} rounding={2}>
+              <Image alt="Botanical art in coral and green" fit="cover" src="https://i.ibb.co/cbjgZft/img-door.jpg" naturalWidth={1} naturalHeight={1}/>
+              <Box position="absolute" top>
+                <Box padding={4}>
+                  <Badge text="Light wash" type="info"/>
+                </Box>
+              </Box>
+            </Mask>
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use established color patterns so users can quickly scan an identify sentiment. However, the badge text should always clearly indicate the sentiment, so color is not the sole indicator of information."
+            defaultCode={`
+            <Table accessibilityLabel="Badge color patterns">
```

Comment:
Changing this to a function component will allow DRYing out the heavily-repeated hardcoded content